pom.xml

```diff
@@ -28,6 +28,17 @@
         <application.attach.zip>true</application.attach.zip>
     </properties>
 
+    <distributionManagement>
```

Comment:
this will not be used right now, this will be later when deploy processes will be specified & set up
please remove this for now

Reply:
done